playwright-e2e/pages/upload-draft-orders.ts

```diff
@@ -0,0 +1,55 @@
+import { type Page, type Locator, expect } from "@playwright/test";
+import { BasePage } from "./base-page";
+
+
+export class UploadDraftOrders extends BasePage {
+
+    constructor(page: Page) {
+        super(page);
```

Comment:
is this test being called/running in pipeline?

Reply:
True